tcti/platformcommand.c

```diff
@@ -62,7 +62,7 @@ TSS2_RC PlatformCommand(
     // Send the command
     iResult = send( TCTI_CONTEXT_INTEL->otherSock, sendbuf, 4, 0 );
     if (iResult == SOCKET_ERROR) {
-        (*printfFunction)(NO_PREFIX, "send failed with error: %d\n", WSAGetLastError() );
+        DebugPrintf(NO_PREFIX, "send failed with error: %d\n", WSAGetLastError() );
```

Comment:
I did rebase, and got some weirdness. But didn’t rebase to your outstanding PR.

What do you want me to do with it?

Will Arthur
Intel Corporation
Server Security Firmware
803-216-2101

From: Philip Tricca [mailto:notifications@github.com]
Sent: Wednesday, April 6, 2016 2:17 PM
To: 01org/TPM2.0-TSS <TPM2.0-TSS@noreply.github.com>
Cc: Arthur, Will C <will.c.arthur@intel.com>
Subject: Re: [01org/TPM2.0-TSS] Added code to clean up when connection between application and resour… (#172)


In tcti/platformcommand.c<https://github.com/01org/TPM2.0-TSS/pull/172#discussion_r58754977>:

> @@ -62,7 +62,7 @@ TSS2_RC PlatformCommand(

>      // Send the command

>      iResult = send( TCTI_CONTEXT_INTEL->otherSock, sendbuf, 4, 0 );

>      if (iResult == SOCKET_ERROR) {

> -        (*printfFunction)(NO_PREFIX, "send failed with error: %d\n", WSAGetLastError() );

> +        DebugPrintf(NO_PREFIX, "send failed with error: %d\n", WSAGetLastError() );

I think this change is an artifact from a rebase? I've got an outstanding PR that gets ride of this function pointer so maybe this snuck in when you were reviewing / testing that work?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/01org/TPM2.0-TSS/pull/172/files/a4bd89f67e2ff748eb24309441334b3da8b54bb9#r58754977>